### DISCUSSION TITLE
Problemas al instalar VUE

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#478] [Q&A]
Estoy tratando de hacer lo mismo que nuestro MC hizo en  el live. Instale NODE, despues instale VUE con el comando "i @vue/cli" , se instalo todo y me aparecio lo siguiente:
![image](https://user-images.githubusercontent.com/99064991/159362601-3cb405d8-42a7-4978-9f54-be9babab1020.png)

Despues me fui a VS para crear un  VUE desde la terminal con el comando "vue create vue-launchx". Pero me dice que no reconoce "vue" como un comando interno o externo.
![image](https://user-images.githubusercontent.com/99064991/159362857-1fa548b7-8395-4d4c-867a-47ef5dd8f9a2.png)

Alguien sabe como se podría solucionar?

**Comment by Ingrzs** [comment]
En la primera imagen pones la i y es -g 
Para que la instalación sea global

**Comment by LuisSantiago10** [comment]
Puedes instalar con npm

**Comment by SandroBorrallez** [comment]
A mi me pasó lo mismo, lo único que hice fue cerrar todas las ventanas de VS y abrí una nueva, y con eso ya pude crear el proyecto, eso es para que el VS actualice la instalación de vue.

Espero y te sirva!!

**Comment by JuanSalvador73** [comment]
Saludos Explorer, ¿Encontraste la solución? Tengo el mismo problema...

**Reply by codingcodax** [reply]
Utiliza `npm init vue@latest`.

**Comment by codingcodax** [comment]
Tienes que ejecutar este comando `npm init vue@latest`.
![image](https://user-images.githubusercontent.com/48135533/160648424-6d8fc322-cc4d-47bc-90ab-6a1a4e2ff5a0.png)
Después solo respondes el formulario, nombre, typescript, etc

**Reply by JuanSalvador73** [reply]
Gracias [codingcodax](https://github.com/codingcodax) sigo con el problema, pero seguiré intentando. :D

**Comment by DanyVeneno** [comment]
Hola  a todos 
este link es donde te pone el comado para instalar el CLI
https://cli.vuejs.org/
![Screenshot_4](https://user-images.githubusercontent.com/89219507/161362387-41017e17-3e9d-4421-923d-ed29ab130138.jpg)
Si ya han trabajado con Yarn es el comando de abajo si no el que tiene la palomita saludos repliquen para saber si se resolvio
Saludos buen fin.

**Comment by alejandronjj** [comment]
Me pasaba lo mismo, me salía eso si lo creaba desde la terminal de visual studio, pero directamente desde cmd sí podría crear el proyecto.
Tuve que instalar la siguiente extensión en visual estudio para que me reconociera los comandos npm
![image](https://user-images.githubusercontent.com/99060336/161400513-83bf029f-0012-45f2-8fa8-89eb8c1c5893.png)
 Para instalarlo, presiona Ctrl+P cuando estés en visual studio y escribe ext install npm script runner
 Después, reinicia VS Code.
 
 Te dejo el link de stackoverflow de donde obtuve la respuesta. Espero que te ayude en caso de que no hayas encontrado solución

https://stackoverflow.com/questions/35563802/running-npm-command-within-visual-studio-code
